pymodels/SI_V24_04/lattice.py

```diff
@@ -526,6 +526,7 @@ def set_vacuum_chamber(the_ring, mode=default_optics_mode):
     other_vchamber = [-0.012, 0.012, -0.012, 0.012]
     idb_vchamber = [-0.004, 0.004, -0.00225, 0.00225]
     ida_vchamber = [-0.012, 0.012, -0.004, 0.004]
+    bc_hfield_vchamber = [-0.004, 0.004, -0.004, 0.004]
```

Comment:
Along the other segments of BC ('low field') the vchamber is equal `bc_vchamber = [-0.012, 0.012, -0.004, 0.004]` or equal  `other_vchamber = [-0.012, 0.012, -0.012, 0.012]`?

Reply:
it is the default chamber @murilobalves

Reply:
fixed!